tests/integration/layers/layers.spec.ts

```diff
@@ -22,11 +22,11 @@ const validPolygon = {
   type: 'Polygon',
   coordinates: [
     [
-      [100, 0],
-      [101, 0],
-      [101, 1],
-      [100, 1],
-      [100, 0],
+      [34.91692694458297, 33.952927285465876],
```

Comment:
please add  it('should return 400 status code for **{each of new info & GPKG checks}')**